openapi/src/main/java/io/helidon/openapi/internal/OpenAPIConfigImpl.java

```diff
@@ -294,6 +297,17 @@ public Builder addServer(String server) {
             return this;
         }
 
+        /**
+         * Sets whether annotation scanning should be disabled.
+         *
+         * @param value new setting for annotation scanning disabled flag
+         * @return updated builder
+         */
+        public Builder scanDisable(boolean value) {
```

Comment:
Consider renaming to `scanningDisabled`.

Reply:
…or `scanningEnabled`, to avoid the double negative situation.

Reply:
I agree with your aesthetics in method naming here. That said, in `OpenAPIConfigImpl` for which this is a builder we are overriding the `scanDisable` method declared in SmallRye's impl. 

I think it's clearer to keep the same method name in the builder.